applications/test/GalleryTemplateTest.scala

```diff
@@ -55,7 +55,7 @@ import scala.jdk.CollectionConverters._
 
   it should "select the largest main picture for the opengraph image when FacebookShareUseTrailPicFirstSwitch is OFF" in {
     FacebookShareUseTrailPicFirstSwitch.switchOff()
-    goTo("/lifeandstyle/gallery/2014/nov/24/flying-dogs-in-pictures") { browser =>
+    goTo("/lifeandstyle/gallery/2014/nov/24/flying-dogs-in-pictures?page=off") { browser =>
```

Comment:
The Play framework runs integration tests with a [`TestBrowser`](https://www.playframework.com/documentation/2.9.x/api/java/play/test/TestBrowser.html) that uses a Selenium WebDriver. Play 2.9 changes the version of Selenium it depends on under the hood from 3.141.59 to 4.14.1. 

In the previous version of the code the tests above could run one after the other without the first affecting the outcome of the second. This is not true anymore, it looks like the [`TestBrowser`](https://github.com/guardian/frontend/blob/6cc188128f816e5b746018a82cabe16de889abcf/common/test/package.scala#L32-L36) we use in our test configuration does not automatically clear its cache after each test so the second test will always fail. We have similar result if we swap the tests. I haven't been able to find a way to do this  programmatically, so we're adding a flag in the url to make sure the cache gets cleared and the tests run independently.

Reply:
Very interesting! I think adding a comment here and on L48 could be helpful to explain the intention with the `page` query parameter. It might also be worth adding a TODO comment to look into programmatically clearing the cache in the future.